docs/tests.md

```diff
@@ -144,22 +144,22 @@ Contributions that add or modify major functionality to the project should typic
 
 ### Python Unit Tests
 
-Our Python unit tests are vanilla flavored [`unittest`](https://docs.python.org/2/library/unittest.html) tests. Unit tests placed in the `tests` directory will be automatically detected by nose&mdash;no manual registration is necessary.
+Our Python unit tests are vanilla flavored [`unit test`](https://docs.python.org/2/library/unittest.html) tests. Unit tests placed in the `tests` directory will be automatically detected by nose&mdash; no manual registration is necessary.
```

Comment:
This one should actually be `unittest` as a single word, since it refers to a Python class.